download-datasets.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# make a datasets directory
+mkdir -p datasets
+
+# move into the datasets directory
+cd datasets
+
+# download the datasets
+wget https://act-bench.s3.us-west-2.amazonaws.com/act-bench-cnn.pkl
+wget https://act-bench.s3.us-west-2.amazonaws.com/act-bench-resnet.pkl
+wget https://act-bench.s3.us-west-2.amazonaws.com/act-bench-vit.pkl
+
+# move back to the root directory
+cd ..
```

Comment:
If this is related to aquasurf, why does it need to be in a separate repo?
Why not provide this script as part of aquasurf?

Reply:
It could be part of aquasurf for convenience, since both come from the same paper.  But they are really separate contributions.  Aquasurf is an algorithm for searching for new activation functions.  Act-Bench-X is a dataset of activation functions and their corresponding accuracies, useful for benchmarking and designing new search algorithms.